tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -229,9 +345,13 @@ function ( $preempt, $args, $url ) use ( $request_url, $response_data, $response
 					return $preempt;
 				}
 
+				if ( is_array( $response_data ) ) {
+					$response_data = json_encode( $response_data );
+				}
```

Comment:
The check here is also no longer necessary, since it'll always be an array. We can unconditionally `json_encode` it like before.